--- a/configure.ac	2023-02-10 22:39:33.568139691 +0100
+++ b/configure.ac	2023-02-25 23:07:05.157696796 +0100
@@ -2102,7 +2102,7 @@
 			 #include <gbm.h>])
 		dnl 17.1.0 is required for gbm_bo_create_with_modifiers
 		PKG_CHECK_EXISTS(gbm >= 17.1.0,
-				 [AC_DEFINE(GBM_BO_WITH_MODIFIERS, 1, [Have gbm_bo_create_with_modifiers])],
+				 [AC_DEFINE(GBM_EXT_BO_WITH_MODIFIERS, 1, [Have gbm_bo_create_with_modifiers])],
 				 [])
 		AC_CHECK_HEADER([gbm.h],
 			[AC_CHECK_LIB(gbm, gbm_bo_map,
